config/app/api/openapi/registry.yml

```
@@ -0,0 +1,117 @@
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/ConfigurationApiRequest'
+            responses:
+                200:
+                    description: 'Expected response to a valid request.'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ConfigurationApiResponse'
+                400:
+                    description: 'Bad request.'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ApiErrorMessages'
+                default:
+                    description: 'Expected response to a bad request.'
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+        put: *paths/private/application-config/post # $ref is not (officially) supported for operation objects
+
+    '/private/application/disconnect':
+        post:
+            summary: 'Disconnects Application from a Tenants store.
+                      Finds Application configuration and removes it.'
+            parameters:
+                -   name: X-Store-Reference
+                    in: header
+                    required: true
+                    description: 'Reference of the Tenants store.'
+                    schema:
+                        type: string
+            responses:
+                204:
+                    description: 'App is disconnected.'
+                400:
+                    description: 'Bad request.'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ApiErrorMessages'
+                default:
+                    description: 'Expected response to a bad request.'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ApiErrorMessages'
+components:
+    schemas:
+        ConfigurationApiResponse:
+            type: object
+            additionalProperties: true
+        ConfigurationApiRequest:
+            properties:
+                data:
+                    type: object
+                    required:
+                        - attributes
+                    properties:
+                        attributes:
+                            type: object
+                            required:
+                                - configuration
+                            properties:
+                                configuration:
+                                    description: 'Contains JSON encoded configuration.'
+                                    type: string
```

```suggestion
                                configuration:
                                    description: 'Contains JSON encoded configuration.'
                                    type: string
                                    example: "{\"configKey1\":\"config value\",\"configKey2\":[\"selectedValue1\",\"selectedValue2\"]}"
```
Let's add the example from the registry service :)